css/avatar-multi-one-button.css

```diff
@@ -0,0 +1,39 @@
+.avatar-multi-one-button .avatar {
+  background-image: url('../images/avatar.jpg');
```

Comment:
@priyanka0901 Avatar image should not be added through CSS. If we will pass it through css will not able change it for dynamic changes.